gunicorn/workers/base_async.py

```diff
@@ -121,8 +121,6 @@ def handle_request(self, listener_name, req, sock, addr):
                     respiter.close()
             if resp.should_close():
                 raise StopIteration()
-        except StopIteration:
```

Comment:
I'm not sure about this. StopIteration is a subclass of Exception, but I don't know this code well, so I'll just defer to @benoitc and @tilgovi.

Reply:
i guess we could add it back as we're going to ignore `try-except-raise`